portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,79 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
+//import com.google.sps.data.Task;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> TestArray = new ArrayList<String>();
```

Comment:
Is this still being used for anything? Or should it be cleaned up?

Reply:
Yes, the "TestArray" is really the array where all comments are stored. I'll change the name to make it more readable in the next commit/pr

Reply:
The Array is added to the DataStoreService

Reply:
On Line 49, I see the comment entity is added to the DataStoreService, but I don't see how CommentArray is involved.

I think you can just delete it, and things should still work.